SPECS/host-installer.spec

```
@@ -269,6 +281,15 @@ done
 rm -f /tmp/firmware-used.$$
 
 %changelog
+* Mon Apr 22 2024 Yann Dirson <yann.dirson@vates.tech> - 10.10.16.xcpng.1-1
+- Loosely rebase on XS's 10.10.16-1, bringing:
```

I had difficulties understanding the changelog, until I understood that 10.10.16-1 was about the packaging, hence this suggestion:
```suggestion
- Loosely rebase packaging on XS's 10.10.16-1, bringing:
```